### PR TITLE
Add print plugin, use npm package via yarn

### PR DESCRIPTION
**Pull request by thet**
- Add the print plugin per default, so that images are loaded before the page is going to be printed.
- Use the npm package and yarn to install lazysizes for a defined way to update / add resources. Via .yarnrc a custom location instead ``node_modules`` is defined.

Fixes https://github.com/collective/collective.lazysizes/issues/50

### CODE REVIEW

File: src/collective/lazysizes/tests/test_upgrades.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def setUp(self):
 
     def test_upgrade_to_2_registrations(self):
         version = self.setup.getLastVersionForProfile(self.profile_id)[0]
-        self.assertGreaterEqual(version, self.to_version)
+        self.assertGreaterEqual(int(version), int(self.to_version))
```
Comment (thet) [inline src/collective/lazysizes/tests/test_upgrades.py]: The string in int cast is necessary. The tests passed before this change for metadata version 9 only by the way strings are compared in python. See below:

Comparison of "10" with other numbers-as-strings:
```
>>> "10" >= "0"
True
>>> "10" >= "1"
True
>>> "10" >= "2"
False
>>> "10" >= "3"
False
>>> "10" >= "4"
False
>>> "10" >= "5"
False
>>> "10" >= "6"
False
>>> "10" >= "7"
False
>>> "10" >= "8"
False
>>> "10" >= "9"
False
>>> "10" >= "10"
True
```

Comparison of "9" with other numbers-as-strings:
```
>>> "9" >= "0"
True
>>> "9" >= "1"
True
>>> "9" >= "2"
True
>>> "9" >= "3"
True
>>> "9" >= "4"
True
>>> "9" >= "5"
True
>>> "9" >= "6"
True
>>> "9" >= "7"
True
>>> "9" >= "8"
True
>>> "9" >= "9"
True
>>> "9" >= "10"
True
```

**Comment by thet** [comment]
travis tests are failing in the moment because of the Plone 4.3 integration.

I don't like having the ``jsregistry.xml`` GS profile file around for Plone 5.x - this is registering the resources into the ``plone-legacy`` bundle - along with the registration of the ``lazysizes`` bundle. That way we end up with a double reigstration of these resources.
I'd actually vote to remove the Plone 4.3 support. Or doing the registration with separate 4.3 and 5.x profiles, as I did for my initial Plone 5.x support PR.

Any comments?

**Comment by rodfersou** [comment]
please, use sc.recipe.staticresources https://github.com/simplesconsultoria/sc.recipe.staticresources and webpack for this (we are migrating all of our addons to use it)

**Comment by hvelarde** [comment]
superseded by #59

**Pull request closed without merging**